.babelrc.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  presets: [
+    [
+      'env',
+      {
+        modules: process.env.ESM ? false : 'commonjs',
+        targets: {
+          node: require('./package.json').engines.node.substring(2), // Strip `>=`
```

Comment:
Until https://github.com/babel/babel/issues/7277 is implemented I think it's better to use explicit string.

Reply:
Why? Then you would not have a single source of truth.

Reply:
Because this construct assumes `>= <version>` and is very brittle.
I personally think `engines` should contain only node version that we run tests against. See my PR for `graphql-js`: https://github.com/graphql/graphql-js/pull/1338/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R21